introduction.md

```diff
@@ -0,0 +1,3 @@
+This is introduction text. I'm writing it here because I have to have something in this file before I can go on to the next section.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.